query/sources.js

```diff
@@ -0,0 +1,10 @@
+/*
+ * Mapping from data sources to type values
+ */
+
+module.exports = {
+  'geonames': ['geoname'],
+  'openaddresses' : ['openaddresses'],
+  'quattroshapes': ['admin0', 'admin1', 'admin2', 'neighborhood', 'locality', 'local_admin'],
+  'openstreetmap' : ['osmaddress', 'osmnode', 'osmway']
+};
```

Comment:
I like the idea of keeping the mappings for source in a separate file. We should refactor the lookup code I wrote in the response commit to use the external lookup, and this file will house the `alias`->`list` as well as `name`->`alias` mappings. No good reason to have in 2 different places.